math/automorphic/isautomorphic.go

```diff
@@ -0,0 +1,35 @@
+// isautomorphic.go
+// description: Checks whether a whole number integer is Automorphic or not. If number < 0 then returns false.
+// details:
+// In mathematics, a number n is said to be a Automorphic number if the square of n ends in the same digits as n itself.
+// ref: (https://en.wikipedia.org/wiki/Automorphic_number)
+// time complexity: O(log10(N))
+// space complexity: O(1)
+// author: [SilverDragonOfR](https://github.com/SilverDragonOfR)
+
+// Package sumofdigits provides method to determine whether a number is automorphic or not.
+package automorphic
```

Comment:
I chose to do so as fibonacci, catalan are there. Should i remove not make a package then?

Reply:
No, move this to `math` package - the reason for them having their own package scope is because there are multiple implementations of those in them (not sure about Catalan but I'm sure about fibonacci) so it was necessary to scope them in their own package but for this standalone package, its not necessary.